### PR TITLE
Create table summarizing topologies, consistency fixes

### PR DESCRIPTION
**Pull request by jasonxian-msft**

### CODE REVIEW

File: pipelines/live/topologies/readme.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+[evr-httpExtension-video-sink](evr-httpExtension-video-sink\topology.json) | Perform event-based recording. When an event of interest is detected by the external AI inference engine via the HTTP extension, those events are published to the IoT Edge Hub. In addition, the events are used to trigger the signal gate processor node which results in the creation of new assets containing clips where event of interest was detected.
+[evr-hubMessage-file-sink](evr-hubMessage-file-sink\topology.json) | Record video clips to the local file system of the edge device whenever an external sensor sends a message to the pipeline topology. The sensor, for example, can be a door sensor.
+[evr-hubMessage-video-sink](evr-hubMessage-video-sink\topology.json) | Use to an object detection AI model to look for objects in the video, and record video clips only when a certain type of object is detected. The trigger to generate these clips is based on the AI inference events published onto the IoT Hub. | [Event-based video recording and playback](https://docs.microsoft.com/azure/azure-video-analyzer/video-analyzer-docs/record-event-based-live-video)
```
Comment (anilmur) [inline pipelines/live/topologies/readme.md:18]: "Use to an object ..."? Edit needed

File: pipelines/live/topologies/readme.md [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+[evr-motion-file-sink](evr-motion-file-sink\topology.json) | Perform event-based recording. When motion is detected from a live video feed, events are sent to a signal gate processor node which opens, sending frames to a file sink node. As a result, new files are created on the local file system of the Edge device, containing the frames where motion was detected. | [Detect motion and record video on edge devices](https://docs.microsoft.com/en-us/azure/azure-video-analyzer/video-analyzer-docs/edge/detect-motion-record-video-edge-devices?pivots=programming-language-csharp)
+[evr-motion-video-sink](evr-motion-video-sink\topology.json) | Perform event-based recording. When motion is detected, those events are published to the IoT Edge Hub. In addition, the motion events are used to trigger the signal gate processor node which will send frames to the video sink node when motion is detected. As a result, new video clips are appended to the video sink containing clips where motion was detected. | [Detect motion, record video to Video Analyzer](https://docs.microsoft.com/en-us/azure/azure-video-analyzer/video-analyzer-docs/edge/detect-motion-record-video-clips-cloud)
```
Comment (anilmur) [inline pipelines/live/topologies/readme.md:20]: As a result, new clips are appended to the Azure Video Analyzer video, corresponding to when motion was detected.

File: pipelines/live/topologies/readme.md [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
+[evr-motion-video-sink-file-sink](evr-motion-video-sink-file-sink\topology.json) | Perform event-based recording of video clips to the cloud as well as to the edge. When motion is detected from a live video feed, events are sent to a signal gate processor node which opens, allowing video to pass through to a file sink node as well as an video sink node. As a result, new files are created on the local file system of the Edge device, and new video clips are appended to your Video Analyzer video file in the cloud. The recordings contain the frames where motion was detected.
```
Comment (anilmur) [inline pipelines/live/topologies/readme.md:21]: ...and new video clips are appended to your Video Analyzer video. The recordings...

File: pipelines/live/topologies/readme.md [hunk 4]
```diff
@@ -1,3 +1,30 @@
...
+[object-tracking](object-tracking\topology.json) | Track objects in a live video feed. The object tracker comes in handy when you need to detect objects in every frame, but the edge device does not have the necessary compute power to be able to apply the vision model on every frame. | [Track objects in a live video](https://docs.microsoft.com/azure/azure-video-analyzer/video-analyzer-docs/track-objects-live-video) 
+[spatial-analysis](spatial-analysis\topology.json) | Live video is sent to an external [spatialAnalysis](https://docs.microsoft.com/azure/cognitive-services/computer-vision/spatial-analysis-operations) module which carries out a supported AI operation. When the criteria defined by the AI operation is met, events are sent to a signal gate processor which opens, sending the frames to a video sink node. As a result, a new clip is appended to the video resource. | [Live Video with Computer Vision for Spatial Analysis](https://aka.ms/ava-spatial-analysis)
```
Comment (anilmur) [inline pipelines/live/topologies/readme.md:30]: As a result, a new clip is appended to the Azure Video Analyzer video resource.

File: pipelines/live/topologies/readme.md [hunk 5]
```diff
@@ -1,3 +1,30 @@
...
+[cvr-with-motion](cvr-with-motion\topology.json) | Capture video and continuously record it to an Azure Video Analyzer video. Additionally, the video from the camera is analyzed for the presence of motion. When motion is detected from a live video feed, relevant inferencing events are published to the IoT Edge Hub.
+[evr-grpcExtension-video-sink](evr-grpcExtension-video-sink\topology.json)  | Perform event-based recording. When an event of interest is detected by the external AI inference engine via the gRPC extension, those events are published to the IoT Edge Hub. In addition, the events are used to trigger the signal gate processor node which results in the creation of new assets containing clips where event of interest was detected.
```
Comment (anilmur) [inline pipelines/live/topologies/readme.md:15]: ... processor node which results in the appending of new clips to the Azure Video Analyzer video, corresponding to when the event of interest was detected.

File: pipelines/live/topologies/readme.md [hunk 6]
```diff
@@ -1,3 +1,30 @@
...
+[evr-httpExtension-video-sink](evr-httpExtension-video-sink\topology.json) | Perform event-based recording. When an event of interest is detected by the external AI inference engine via the HTTP extension, those events are published to the IoT Edge Hub. In addition, the events are used to trigger the signal gate processor node which results in the creation of new assets containing clips where event of interest was detected.
```
Comment (anilmur) [inline pipelines/live/topologies/readme.md:16]: Same as before - "assets" are for LVA...

**Review by anilmur — CHANGES_REQUESTED** [review]
Thanks for this PR. Some edits suggested.

I have pointed out some errors in the text in the overall readme.md, please propagate to the individual readme files.

**Comment by jasonxian-msft** [comment]
Closing this PR to make updates

**Pull request closed without merging**